### PR TITLE
OSIRAA-0.6

### PR DESCRIPTION
**Pull request by JohnSzingerCR**

### CODE REVIEW

File: drp_aa_mvp/data_rights_request/views.py [hunk 1]
```diff
@@ -302,6 +306,35 @@ def get_request_actions_form_display (covered_biz):
...
+        "relationships": [ ],
+        # callbackk url for the AA that the CB can hit to provide status updates, NYI
+        "status_callback": "https://dsr-agent.example.com/update_status",           
+        
+        # 3
+        # claims in IANA JSON Web Token Claims page
+        # https://www.iana.org/assignments/jwt/jwt.xhtml#claims
+    }
+
+    # todo: - where does identity payload fit into the new scheme ... ?
```
Comment (rrix) [inline drp_aa_mvp/data_rights_request/views.py]: the claims in the identity payload would be included in the root json object directly, under # 3 above; you could just modify `create_id_payload` to not return `iss` and `sub` claims and merge the result of that in to this dict.

**Review by rrix — COMMENTED** [review]
on the right track, looks good; addressed your question inline

File: drp_aa_mvp/data_rights_request/views.py [hunk 2]
```diff
@@ -86,17 +88,18 @@ def send_request_excercise_rights(request):
     user_identity   = IdentityUser.objects.get(pk=user_id_id)
     request_action  = request.POST.get('request_action')
     covered_regime  = request.POST.get('covered_regime')
-    request_url     = covered_biz.api_root_endpoint + "/exercise"
+    request_url     = covered_biz.api_root_endpoint + f"/exercise?kid={auth_agent_drp_id}"
```
Comment (rrix) [inline drp_aa_mvp/data_rights_request/views.py:96]: mostly a nitpick but, considering using [urlparse.parse.urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) here + elsewhere when generating the `request_url`s

**Review by rrix — APPROVED** [review]

File: drp_aa_mvp/data_rights_request/views.py (hunk 2 above)
Comment (JohnSzingerCR) [reply drp_aa_mvp/data_rights_request/views.py:96]: This is a good idea.  I'll fold it into 0.7

**Pull request merged**